pkg/target/config.go

```diff
@@ -0,0 +1,59 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package target
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Gardens []Garden `yaml:"gardens"`
```

Comment:
could you add a config option so that kubeconfigs are cloned when targeting?

See also https://github.com/gardener/gardenctl/issues/499#issuecomment-749380248 "Use case b"
> I'm targeting some control plane and now want to analyze another control plane on the same seed side-by-side in multiple panes.
What I currently do for this is "cloning" my kubeconfig, which allows me to use kubens to have to panes targeting the same cluster but different namespaces.

> [...]
Therefore, I am clearly for option #3. Every time you target something or retarget it, it's another "instance" of the kubeconfig.

Reply:
as discussed we will takle this later..